### PR TITLE
[PBW-4382] aspect ratio calculation for flash and HLS

### PR DESCRIPTION
**Pull request by ksunika**

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -1073,7 +1073,16 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     //get original video width/height dimensions
     getIntrinsicDimensions: function() {
       var video = this.state.mainVideoElement.get(0);
-      this.state.mainVideoAspectRatio = this.calculateAspectRatio(video.videoWidth, video.videoHeight);
+      var width, height;
+      if (typeof video.videoWidth != 'undefined'){
+        width = video.videoWidth;
+        height = video.videoHeight;
+      }
+      else {
```
Comment (mistermendez) [inline js/controller.js]: I should have mentioned this before - can we make the default HTML5?

Check for flash and if no flash default to html5.

Comment (ksunika) [inline js/controller.js]: error occurs if I try to call video.TGetProperty("/", 8) on html5-video

**Comment by ksunika** [comment]
waiting for PBF side of this to be done. @nagavenivn will give us an update.

**Comment by mistermendez** [comment]
Try using this PBF branch to test - https://github.com/ooyala/html5-video-plugins/tree/PBF-488

**Comment by mistermendez** [comment]
@ksunika looks good

I added live stream to the mix

please review and merge if it looks good to you

File: js/controller.js [hunk 2]
```diff
@@ -1163,7 +1163,24 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     //get original video width/height dimensions
     getIntrinsicDimensions: function() {
       var video = this.state.mainVideoElement.get(0);
-      this.state.mainVideoAspectRatio = this.calculateAspectRatio(video.videoWidth, video.videoHeight);
+      var liveStreamDimension = (this.state.authorization.streams && this.state.authorization.streams[0].aspect_ratio) ? Utils.reformatAspectRatio(this.state.authorization.streams[0].aspect_ratio) : null;
```
Comment (ksunika) [inline js/controller.js]: @mistermendez, I added the check for streams here. The rest looks good to me.

**Comment by mistermendez** [comment]
@ksunika  added try/catch block

**Comment by ksunika** [comment]
@mistermendez, looks good

**Pull request merged**